src/clojure_ttt/computer.clj

```diff
@@ -0,0 +1,36 @@
+(ns clojure-ttt.computer
+  (use [clojure-ttt.board]))
+
+  (defn- score [board depth marks]
+    (cond (and (any-wins? board) (= (last-move board) (first marks))) depth
```

Comment:
As you've done below this, I believe it's more idiomatic to have 'cond' like this:
(cond
  (condition1?) if
  :else else)